src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -40,15 +40,22 @@ export class GraphStyles {
         css: {
           // color: PfColors.Black,
           content: (ele: any) => {
+            const service = ele.data('serviceName') || ele.data('service').split('.')[0];
+            const namespace = ele.data('namespace');
             const version = ele.data('version');
+
+            if (ele.data('isOutside')) {
```

Comment:
I think this should be *after* the line 53
```
if (!ele.data('showNodeLabels')) {
   return '';
}
```

else the label will be showing even though showNodeLabels is false

Reply:
True! Fixed.